lepton/starterkits/model_compression.ipynb

```diff
@@ -0,0 +1,701 @@
+    "\n",
+    "## Requirements\n",
+    "\n",
+    "### System Configuration\n",
+    "- Access to at least 8 NVIDIA GPUs, each with a memory of at least 80GB (e.g., 8 x H100-80GB or 8 x A100-80GB).\n",
+    "- A Docker-enabled environment, with [NVIDIA Container Runtime](https://developer.nvidia.com/container-runtime) installed, which will make the container GPU-aware.\n",
+    "\n",
+    "Get your Hugging Face [access token](https://huggingface.co/docs/hub/en/security-tokens), which will be used to download gated models or datasets.\n",
+    "\n",
+    "**NOTE:** The default configuration in the notebook runs on 8 x 80GB NVIDIA GPUs. However, you can potentially reduce the Tensor Parallel size (`TENSOR_PARALLEL_SIZE`) along with the Micro-Batchsize (`MICRO_BATCH_SIZE`) in the distillation scripts to accommodate lower resource availability.\n",
+    "\n",
+    "## Create a Pruned and Distilled Model with NeMo Framework\n",
+    "\n",
+    "For pruning and distilling the model, you will use the NeMo Framework, which is available as a [Docker container](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo). These notebooks has been tested on `nvcr.io/nvidia/nemo:25.09` container with `nvidia-modelopt==0.35.1`.\n",
+    "\n",
+    "1. Run the container using the following command. You will mount your local directory to `/workspace` so the model and dataset will be stored in a persistent location. If you are using your own model and dataset, you can change the paths in the notebooks accordingly.\n",
+    "\n",
+    "```bash\n",
+    "export FW_VERSION=25.09\n",
+    "```\n",
+    "\n",
+    "```bash\n",
+    "docker run \\\n",
+    "  --gpus all \\\n",
+    "  --shm-size=16GB \\\n",
+    "  --net=host \\\n",
+    "  --ulimit memlock=-1 \\\n",
+    "  --rm -it \\\n",
+    "  -v ${PWD}:/workspace \\\n",
+    "  -w /workspace \\\n",
+    "  nvcr.io/nvidia/nemo:$FW_VERSION bash\n",
+    "```\n",
+    "\n",
+    "2. From within the container, copy the notebooks to your local directory so changes remain persistent (only if running first time).\n",
+    "\n",
+    "```bash\n",
+    "cp -r /opt/NeMo/tutorials/llm/qwen/pruning-distillation/* /workspace\n",
+    "```\n",
+    "\n",
+    "3. From within the container, login with your Hugging Face token to download any gated models or datasets (not required if you have already downloaded the model and datasets).\n",
+    "\n",
+    "```bash\n",
+    "huggingface-cli login --token <YOUR_HF_ACCESS_TOKEN>\n",
+    "```\n",
+    "\n",
+    "4. Start the Jupyter lab:\n",
+    "\n",
+    "```bash\n",
+    "pip install --upgrade ipywidgets notebook\n",
+    "jupyter lab --ip 0.0.0.0 --port=8888 --allow-root\n",
+    "```\n",
+    "\n",
+    "5. Then, navigate to this directory which contains a list of notebooks that cover all the steps to create a distilled 6B model from Qwen3-8B.\n",
+    "\n",
```

Comment:
This will all be taken care of by the Starter Kits feature on DGX Cloud Lepton, so this section can be removed.